RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -61,7 +68,9 @@
 #   hcalLaserEventFilter *
    EcalDeadCellTriggerPrimitiveFilter* 
 #   *goodVertices * trackingFailureFilter *
-   eeBadScFilter
+   eeBadScFilter *
 #   ecalLaserCorrFilter *
 #   trkPOGFilters
+   chargedHadronTrackResolutionFilter *
```

Comment:
changes in this file should not be made in 74X, until changes in RECO/miniAOD workflows are allowed.